src/collections/_documentation/performance/performance-tab.md

```diff
@@ -0,0 +1,46 @@
+
+The Throughput graph indicates the number of transactions over a given time range. Users can quickly visualize the traffic patterns in their application and see if a spike in throughput might be correlated with a dip in their Apdex score.
+
+## Transaction Table
+Underneath the graphs, users can view a list of their transactions with the capability to filter down on which transactions users want to view. In this table, users can sort on the provided columns to understand key characteristics of their transactions like [LINK: P95] duration time, [LINK: Average Transaction Duration] time, or [LINK: Throughput].
+
+## My Key Transactions
+"My Key Transactions" are transactions that users bookmark and can view in a separate dedicated tab. Users may want to bookmark a Transaction as a "Key Transaction" if they monitor performance on a particular transaction or group of transactions. Users can set a Transaction as a "Key Transaction" on the Transaction Summary page. The "All Transactions" view also displays these transactions.
+
+## Transaction Summary
+
+[ SCREENSHOT of Transaction Summary page ]
+
+Navigate to the Transaction Summary page by clicking on any Transaction on the Performance Homepage. On the Transaction Summary page, users can find graphs, a list of slowest transactions, facet maps, and summary statistics about the transaction.
+
+## Transaction Summary Graphs
+Duration Breakdown: By graphing P50, P75, P95, P99, and P100 durations distinctly, users can utilize this display to see their transaction performance over time. Users can toggle the display of each segment by clicking on the legend (including Releases). Also, users can zoom in on specific slices to investigate spikes or possible performance regressions from a release.
+
+Latency Histogram: Users can visualize the frequency of transactions that occur within each response time segment. This histogram view can help users understand the shape of their transaction response times and quickly see outliers. Users can click on specific segments to drill down for more granularity.
+
+Apdex / Throughput Graphs: Users can see the same graphs from the Performance homepage isolated to a specific transaction. Users can also zoom in here to investigate spikes and see potential correlations between Apdex and throughput.
+
+Duration Percentiles: This graph shows the average transaction duration across distinct percentiles. For example, users can see how different the P50 and P99 response time durations are for a given transaction and understand the rate of increase between segments.
+
+## Sidebar
+The sidebar contains a Tag Summary and more statistics about the transaction. This information updates dynamically if a user changes any of the selections in the global header or when a user drills in on a latency segment (applicable when viewing the Latency Histogram).
```

Comment:
We are working on moving the apdex, throughput and error rate graphs into the sidebar.

Reply:
Will apdex, throughput, and error rate graphs be available by 05/13? Was hoping to launch docs by that date.

Reply:
I hope so. I have the pull request open now in getsentry/sentry#18680